http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriberSpec.scala

```diff
@@ -1147,7 +1148,7 @@ class LeoPubsubMessageSubscriberSpec
 
   // delete app and not delete disk when specified
   // update app status and disk id
-  it should "delete app and not delete disk when specified" in isolatedDbTest {
+  it should "delete app and not delete disk when diskId not specified" in isolatedDbTest {
```

Comment:
Would it be possible to also verify that the call to delete the bucket is not made in this case?